components/invitation_modal/invitation_modal_guests_step.jsx

```diff
@@ -58,18 +58,32 @@ export default class InvitationModalGuestsStep extends React.Component {
         this.props.onEdit(this.state.usersAndEmails.length > 0 || this.state.channels.length > 0 || e.target.value !== '');
     }
 
-    usersLoader = async (term) => {
+    debouncedSearchProfiles = debounce((term, callback) => {
+        this.props.searchProfiles(term).then(({data}) => {
+            callback(data);
+            if (data.length === 0) {
+                this.setState({termWithoutResults: term});
+            } else {
+                this.setState({termWithoutResults: null});
+            }
+        });
+    }, 150);
+
+    usersLoader = (term, callback) => {
         if (isEmail(term)) {
-            return [];
+            callback([]);
+            return;
+        }
+        if (this.state.termWithoutResults && term.startsWith(this.state.termWithoutResults)) {
+            callback([]);
```

Comment:
The callback is the way to populate the react-select options, in this case, is going to be an empty list. This function is call for every single text change of the input field, so... is expected that the callback is called every single time. There is not problem on call the callback as many times as we want, because the last one is the important one, the one that updates the options in the selector.